src/generic/generic-game-factory.vala

```diff
@@ -0,0 +1,54 @@
+// This file is part of GNOME Games. License: GPLv3
```

Comment:
Check that for all your commit messages and anytime you edit one (maybe I didn't point you to this convention earlier).